R/select.R

```diff
@@ -16,6 +17,9 @@ select.duckplyr_df <- function(.data, ...) {
   rel_try(list(name = "select", x = .data, args = try_list(dots = enquos(...))),
     # We could count and create a zero-col data frame, but we can't
     # create a duckplyr frame from it anyway.
+    #' @section Caveats:
```

Comment:
Maybe this section should be called Fallbacks?

Reply:
Oh and we could have an automatic sentence about telemetry in all fallback section later once telemetry is easier to tweak?